.github/workflows/ci.yml

```
@@ -18,78 +18,83 @@ concurrency:
       - uses: actions/checkout@v4
-
-      - name: Restore environment
+      
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: "20"
+          
+      - name: Cache dependencies
+        id: npm-cache
         uses: actions/cache@v3
```

actionlint (1.7.7)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)